ProductsService/ProductsService.Application/Validators/Categories/UpdateCategoryRequestValidator.cs

```diff
@@ -0,0 +1,29 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using ProductsService.Application.UseCases.CategoryUseCases.Commands.UpdateCategory;
+
+namespace ProductsService.Application.Validators.Categories;
+
+public class UpdateCategoryRequestValidator : AbstractValidator<UpdateCategoryRequest>
+{
+    public UpdateCategoryRequestValidator()
+    {
+        RuleFor(req => req.Name)
+            .NotEmpty()
+            .WithMessage("Wrong category name");
+
+        RuleFor(req => req.CategoryId)
+            .GreaterThan(0)
+            .WithMessage("Wrong category id");
+
+        RuleFor(req => req.Image)
+            .Must(BeAnImage)
+            .WithMessage("You should upload only image files")
+            .When(req => req.Image is not null);
+    }
+
+    private bool BeAnImage(IFormFile? file)
```

Comment:
the same

Reply:
It's used as parameter of FluentValidation Must method, so i thought it's better to call it BeAnImage, to get something like Must(BeAnImage) when i call it